SEED-platform/seed
# Fixed ng-cloak issues and linux documentation

#### What's this PR do?
- Fixes ng-cloak issues
- Updates linux documentation with a variety of improvements
- Further npm dependency improvements
- Lint
#### How should this be manually tested?
For code review ignore whitespace: https://github.com/SEED-platform/seed/pull/1803/files?w=1

Refresh the page, check that you don't briefly see an un-rendered angular template
#### What are the relevant tickets?
#1304
